src/components/Compare/columns.tsx

```diff
@@ -115,8 +114,7 @@ class VaccinationsColumn extends MetricColumn {
           </VaccinationsCellValue>
           <VaccinationsCellProgressBar>
             <VaccineProgressBar
-              vaccinationsCompleted={vaccinationsCompleted}
-              vaccinationsInitiated={vaccinationsInitiated}
+              vaccinationsRatio={vaccinationsInitiated}
```

Comment:
I think we should rename `vaccinationsInitiated`.  Maybe just `vaccinations` or `vaccinationsRatio` so that we don't need to rename the variable every time there's a new booster.